css/styles.css

```diff
@@ -190,3 +216,61 @@ Colors:
 .btn-white:active {
   background-color: rgba(255, 255, 255, 0.35);
 }
+
+/* ***************** */
+/* About */
+/* ***************** */
+.about {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 71rem;
+  padding-left: 26rem;
+  padding-right: 24rem;
+  background-image: linear-gradient(
+      to right bottom,
+      rgb(255 255 255 /14%),
+      rgb(0 0 0 /10%)
+    ),
+    url("../img/bg-about.png");
+  box-shadow: inset 0 0 2.4rem 1.2rem rgb(0 0 0 /6%);
+}
+
+.about-content {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  flex: 1;
```

Comment:
`flex` requires 3 properties set. btw property looks redundant here. Why was it add?